src/main/scala/org/broadinstitute/dsde/firecloud/model/MethodRepository.scala

```diff
@@ -62,9 +62,9 @@ object MethodRepository {
     val ListAll = List("All")
   }
 
-  def validateEmail(email:String): Boolean = {
+  def validatePublicOrEmail(email:String): Boolean = {
     // TODO: real email address validation!
-    !email.trim.isEmpty
+    !email.trim.isEmpty || email.trim.equalsIgnoreCase("public")
```

Comment:
I really didn't like getting rid of the intention behind the method so I added an email check and tests to cover the different cases.